packages/core/admin/admin/src/translations/en.json

```diff
@@ -481,26 +481,26 @@
   "components.FilterOptions.FILTER_TYPES.$notNull": "is not null",
   "components.FilterOptions.FILTER_TYPES.$null": "is null",
   "components.FilterOptions.FILTER_TYPES.$startsWith": "starts with",
-  "components.Input.error.attribute.key.taken": "This value already exists",
+  "components.Input.error.attribute.key.taken": "The value already exists",
   "components.Input.error.attribute.sameKeyAndName": "Can't be equal",
   "components.Input.error.attribute.taken": "This field name already exists",
   "components.Input.error.contain.lowercase": "Password must contain at least one lowercase character",
   "components.Input.error.contain.number": "Password must contain at least one number",
   "components.Input.error.contain.uppercase": "Password must contain at least one uppercase character",
-  "components.Input.error.contentTypeName.taken": "This name already exists",
+  "components.Input.error.contentTypeName.taken": "The name already exists",
   "components.Input.error.custom-error": "{errorMessage} ",
-  "components.Input.error.password.noMatch": "Passwords do not match",
-  "components.Input.error.validation.email": "This is an invalid email",
-  "components.Input.error.validation.json": "This doesn't match the JSON format",
+  "components.Input.error.password.noMatch": "The passwords do not match",
+  "components.Input.error.validation.email": "The value is an invalid email",
+  "components.Input.error.validation.json": "The value is invalid JSON",
   "components.Input.error.validation.lowercase": "The value must be a lowercase string",
-  "components.Input.error.validation.max": "The value is too high.",
-  "components.Input.error.validation.maxLength": "The value is too long.",
-  "components.Input.error.validation.min": "The value is too low.",
-  "components.Input.error.validation.minLength": "The value is too short.",
-  "components.Input.error.validation.minSupMax": "Can't be superior",
-  "components.Input.error.validation.regex": "The value does not match the regex.",
-  "components.Input.error.validation.required": "This value is required.",
-  "components.Input.error.validation.unique": "This value is already used.",
+  "components.Input.error.validation.max": "The value is larger than the maxmimum",
+  "components.Input.error.validation.maxLength": "The value is longer than the maximum",
+  "components.Input.error.validation.min": "The value is smaller than the minimum",
+  "components.Input.error.validation.minLength": "The value is shorter than the minimum",
+  "components.Input.error.validation.minSupMax": "The value can not be superior",
+  "components.Input.error.validation.regex": "The value does not match the regular expression",
```

Comment:
Might be a silly question but is there a more human way to talk about regex in your opinion? Cause I feel like for a non-tech this won't mean anything 🤔

Reply:
We could say "pattern"?

Reply:
A common complaint around this message specifically is we don't say what the regex is either

Reply:
I agree it's madness, I would make it what we're actually looking for e.g can't use underscore

Reply:
I've created a feature request for that - @derrickmehaffy thanks for sharing. 🙏🏼  Whenever something about that comes up could you please redirect users there? https://feedback.strapi.io/customization/p/fields-with-regex-patterns-should-display-the-required-pattern-in-the-error-mess

> I agree it's madness, I would make it what we're actually looking for e.g can't use underscore

I think users configuring the content-type can set this up already using descriptions. Otherwise we'd an AI to interpret the regex and make it human-readable.